src/google_takeout_metadata/processor_batch.py

```
@@ -249,6 +261,18 @@ def process_directory_batch(root: Path, use_localtime: bool = False, append_only
                 meta.albums.extend(find_albums_for_directory(fixed_json_path.parent))
             
+            # Organisation des fichiers si demandée
+            if file_organizer and (meta.archived or meta.trashed or meta.locked):
+                try:
+                    moved_media, moved_json = file_organizer.move_file_with_sidecar(fixed_media_path, fixed_json_path, meta)
+                    if moved_media and moved_json:
+                        # Mettre à jour les chemins pour la suite du traitement
+                        fixed_media_path = moved_media
+                        fixed_json_path = moved_json
+                        logger.info(f"📁 Fichier organisé : {media_path.name} → {moved_media.parent.name}/")
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)